TheWarWithin/HunterMarksmanship.lua

```diff
@@ -886,7 +886,7 @@ spec:RegisterAbilities( {
         handler = function ()
             if buff.lock_and_load.up then removeBuff( "lock_and_load" )
             elseif buff.secrets_of_the_unblinking_vigil.up then removeBuff( "secrets_of_the_unblinking_vigil" ) end
-            if talent.precise_shots.enabled then applyBuff( "precise_shots" ) end
+            if talent.precise_shots.enabled then applyBuff( "precise_shots", 15, 2, nil ) end
```

Comment:
`applyBuff( "precise_shots", nil, 2 )` -- args are `aura[, duration[, stacks ] ]`

`duration` defaults to `class.auras.precise_shots.duration` if `nil`/omitted.

Reply:
Perfect, change implemented.